src/App.js

```diff
@@ -151,8 +151,8 @@ class App extends Component {
               />
               <div className="filters">
                 <button onClick={this.showAllParks}>Show All Parks</button>
-                <button onClick={this.showVisitedParks}><i class="fas fa-hiking" id="partOfList"></i> Show {this.state.visitedParkCodes.length} Visited Parks</button>
-                <button onClick={this.showBucketList}><i class="fas fa-clipboard-list" id="partOfList"></i> Show {this.state.bucketListParkCodes.length} Bucket List Parks</button>
+                <button onClick={this.showVisitedParks}><img className="loc-icons" src="./assets/marker-icon-green.png" alt="green icon" /> Show {this.state.visitedParkCodes.length} Visited Parks</button>
```

Comment:
The purple and green icons look great on the map and I like how cohesive it looks with them on the view buttons also